### PR TITLE
feature: ZENKO-382 skip version for nfs exported bucket

### PR DESCRIPTION
**Pull request by rahulreddy**

### CODE REVIEW

**Comment by rahulreddy** [comment]
Sorry no tests due to urgency of the PR

**Comment by rahulreddy** [comment]
@bert-e wait

**Review by ploki — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e help

**Comment by rahulreddy** [comment]
@bert-e bypass_peer_approval

**Comment by rahulreddy** [comment]
@bert-e bypass_author_approval

File: lib/api/bucketPutVersioning.js [hunk 1]
```diff
@@ -133,6 +138,10 @@ function bucketPutVersioning(authInfo, request, log, callback) {
     ], (err, bucket) => {
         const corsHeaders = collectCorsHeaders(request.headers.origin,
             request.method, bucket);
+        if (err && err.code === 'NFSBUCKET') {
+            log.trace('skipping versioning for nfs exported bucket');
+            return(null, corsHeaders);
```
Comment (jonathan-gramain) [inline lib/api/bucketPutVersioning.js]: I think you forgot `callback(...)`

**Review by jonathan-gramain — CHANGES_REQUESTED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**